src/css/main.css

```diff
@@ -91,3 +92,16 @@ nav.scrolled {
   @apply shadow-2xl;
   border-bottom: 0px;
 }
+
+.login-size {
+  @apply max-w-sm;
+}
+
+.empty-cart {
```

Comment:
These classes fixed all the css issues on heroku. Eventually I will refactor all css to custom classes